src/lib/Makefile.am

```diff
@@ -18,6 +18,9 @@
 
 SUBDIRS = include
 
+AM_LIB_CFLAGS = -I$(top_srcdir)/src/lib/include -I$(top_srcdir)/src -I$(top_srcdir)/config \
+        -fPIC -shared -fvisibility=hidden
```

Comment:
Even if those flags are here, we should check availability first by `AX_CHECK_LINK_FLAG`